routes/auth.js

```diff
@@ -81,4 +80,13 @@ function validateLogin(req) {
   return Joi.validate(req, schema);
 }
 
+
+// Method for generating the token
+function generateAuthToken (_id) {
```

Comment:
Reason for moving this here is because we don't want to do anything other than DB operations inside models, we also need to abstract this more and create a separate controller for this in future.

Reply:
That's fine. As long as it still works 👍🏽